### DISCUSSION TITLE
Node not stating status

### DISCUSSION THREAD

**Discussion by FixInProduction** [zachowj/node-red-contrib-home-assistant-websocket#1626] [Q&A] [answered]
I just updated to 18.1.1 (4.0.3) and I no longer have node status under the node. I have also noticed that Debug nodes no longer capture information and have no output. The strange part is everything is working in my flows, just nothing is being updated in the GUI. I have tried restarting Node Red and still the same.

<img width="517" alt="image" src="https://github.com/user-attachments/assets/e581096a-d4c7-45fd-afe5-649380a13a23">

Any thoughts or is this a bug on the new version?

**Comment by zachowj** [comment, marked as answer]
WebSocket communication appears to be rejected when accessing Node-RED through an iframe in the Home Assistant UI rather than directly via port 1880. This issue affects inject nodes, the debug window, and the new issues feature, preventing proper communication between the Node-RED backend and UI.

Until this is resolved or the root cause for the WebSocket issue is identified, I recommend accessing Node-RED directly on port 1880.

Related issues for tracking:
- https://github.com/hassio-addons/addon-node-red/issues/1938
- https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/1480

**Reply by FixInProduction** [reply]
That would make sense, thanks for the heads up and tip!

**Reply by alexruffell** [reply]
While it is not 100% of the time, I have this issue daily. Is a fix on the horizon?
